### ISSUE TITLE
Conditional compilation support was broken in 1.6.2

### ISSUE THREAD

**Issue by Gepar** [zabel-xyz/plsql-language#58]
In current version (1.6.2) parser stops parsing file when meets some conditional compilation directives.
Not in all cases, but very often. In previous version parser just ignore them.
You can see it in outline.

Example in package body:
![capture](https://user-images.githubusercontent.com/5103892/48468083-9359b880-e7f3-11e8-8e42-a39c93377e04.PNG)

`create or replace package body is_there_anybody_out_there as

    procedure test0 is begin null; end;

    /* YOU SHALL NOT PASS */
    procedure test1 is
    begin
        while false loop
            begin
                $IF true $THEN
                    null;
                $END
            exception when others then
                null;
            end;
        end loop;
    end;

    procedure test2 is begin null; end;
    procedure test3 is begin null; end;

end;```

Example in package header:
![capture2](https://user-images.githubusercontent.com/5103892/48468646-221b0500-e7f5-11e8-9728-dc7ba27f5a63.PNG)
```create or replace package is_there_anybody_out_there as

    procedure test0;

    --it's not the procedure  you're looking for
    $IF true $THEN
    procedure test1;
    $END

    procedure test2;
    procedure test3;

end;```

**Comment by Gepar** [comment]
Maybe the best way for now it's just to ignore them.
But i can't rewrite regexp to do this:
`RegExpParser.regComment = (?:\\/\\*[\\s\\S]*?\\*\\/)|(?:--.*);`

**Comment by zabel-xyz** [comment]
Conditional compilation has never been supported.

In spec, it has never work, because regexp consider only definition ending with character ;
`regSpecSymbols = `(?:(${RegExpParser.REG_WORD}+)\\s+(${RegExpParser.REG_WORD}+)\\s*(?:\\s*;|.[^;]*;))`
`
In body, it was ignored before

I'll modify the regExp parser to support it...

**Comment by zabel-xyz** [comment]
Fix in version 1.6.3
I added some tests, see [xyz_myPackage.sql](https://github.com/zabel-xyz/plsql-language/blob/master/main/test/sql/xyz_myPackage.sql)
